### PR TITLE
Port AbstractJettyBundleActivator to new config

### PR DESCRIPTION
**Pull request by bbaldino**
A couple notes on this:

1. Technically this PR contains a breaking change: previously, if the `keyStorePath` value was not an absolute path, `AbstractJettyBundleActivator` would attempt to resolve it relative to the user's home directory and, if the user's home directory couldn't be determined, the current directory.  I removed that logic and it's now required to be an absolute path (**even for the legacy config file**).  I didn't replicate the old behavior because it was leveraging some features of the legacy `ConfigurationService`.  I did check our infra code and saw that we always set it as an absolute path.

1. Jicoco defined defaults for some of these values before and I preserved those, but I'm not sure it makes sense for them to stay there (as opposed to living in `reference.conf`)

### CODE REVIEW

**Comment by bbaldino** [comment]
jenkins, test this please

**Comment by JonathanLennox** [comment]
Do the default Debian installations also set the absolute path (if they set this at all)?

If so, I think this is fine - the documentation always seems to list this as either
```
org.jitsi.videobridge.rest.jetty.sslContextFactory.keyStorePath=/etc/jitsi/videobridge/ssl.store
```
or
```
org.jitsi.videobridge.rest.private.jetty.sslContextFactory.keyStorePath=/path/to/keystore
```

so I don't think we've ever documented that a relative path would work.

**Comment by bbaldino** [comment]
> keyStorePath

I don't see it referenced at all in the jvb repo (outside the docs), so I don't think the debian install sets it at all.

File: jicoco/src/main/kotlin/org/jitsi/rest/JettyBundleActivatorConfig.kt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        "$newPropertyPrefix.jetty.need-client-auth".from(JitsiConfig.newConfig)
+        "default" { false }
+    }
+
+    /**
+     * The port on which the Jetty server is to listen for HTTPS requests
+     */
+    val tlsPort: Int by config {
+        "$legacyPropertyPrefix.jetty.tls.port".from(JitsiConfig.legacyConfig)
+        "$newPropertyPrefix.jetty.tls-port".from(JitsiConfig.newConfig)
```
Comment (bgrozev) [inline jicoco/src/main/kotlin/org/jitsi/rest/JettyBundleActivatorConfig.kt:80]: Do we need the ".jetty." part in the new properties?

Comment (bbaldino) [reply jicoco/src/main/kotlin/org/jitsi/rest/JettyBundleActivatorConfig.kt:80]: I'll remove

**Comment by bbaldino** [comment]
jenkins, test this please

**Pull request merged**